dask/array/core.py

```diff
@@ -1047,6 +1047,19 @@ def _set_chunks(self, chunks):
     def __len__(self):
         return sum(self.chunks[0])
 
+    def __array_ufunc__(self, numpy_ufunc, method, *inputs, **kwargs):
+        import pdb; pdb.set_trace()
+        from . import ufunc
+        da_ufunc = getattr(ufunc, numpy_ufunc.__name__)
+        if method == '__call__':
+            return da_ufunc(*inputs, **kwargs)
```

Comment:
I would suggest writing just `elemwise(numpy_ufunc, *inputs, **kwargs)`, once you've verified `numpy_ufunc.signature is None` (which means that isn't a gufunc). This would allow you to handle arbitrary ufuncs without dask wrappers.